### DISCUSSION TITLE
Welcome to HP ProDesk 400 G1 Hackintosh every macOS Discussions!

### DISCUSSION THREAD

**Discussion by HJebbour** [HJebbour/HP-ProDesk-400-G1-Hackintosh-every-macOS#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by 0x8008** [comment]
Amazing job.

Now I'm sad I removed my old gh account with the Catalina repo in it, I'd be mentioned as well lol
